assets/js/modules/adsense/components/common/SiteSteps.js

```diff
@@ -44,7 +44,7 @@ export default function SiteSteps() {
 		},
 		{
 			linkText: __(
-				'Verified that your site is marked as "Ready"',
+				'Verify that your site is marked as "Ready"',
```

Comment:
Using the past tense form is correct here in the context it is used:

![image](https://user-images.githubusercontent.com/1621608/212500017-645c83b1-a0df-4cbe-a640-fb35235a2dcb.png)